jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -92,6 +92,25 @@ public boolean errorPageForMethod(String method)
         return ERROR_METHODS.contains(method);
     }
 
+    /**
+     * Write an error response.
+     * <p>
+     * In Servlet implementations of the {@link ErrorHandler}, this method is overridden to signal that
+     * a sendError should be triggered a when it enters the servlet channel.
```

Comment:
```suggestion
     * a sendError should be triggered a when it enters the servlet channel.
     * The default implementation calls {@link Response#writeError(...)}.
```

The link is not correct, but you get the idea